editor/protobuf/js/binary/constants.js

```diff
@@ -0,0 +1,376 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Is there any reason to check these in rather than requiring the user to download the source? I'm not super excited about bloating the repo with vendored third party deps or worrying about keeping them in sync with HEAD.

Reply:
The only reason to check these in is to permit the automated test.

If it could be sufficient for the user to download the protobuf JS runtime, then they could download a protoc binary at the same time and we could abandon this PR.

Could the need for this JS runtime complicate automated testing even after presence detection is released? Are we barking up the wrong tree?

Reply:
We can download arbitrary files during the automated tests (and we already download quite a lot of things in the current tests), so I think I'd prefer not including "test-only" files in the repo if there will be any associated maintenance burden. 

That said, if checking in this small set of extra files will make it much easier for _users_ to spin up the editor, that might be worth it. In that case, should we consider creating an "editor/third_party" folder to clearly separate these files?

Reply:
As discussed, I have reverted the added protobuf dependencies and published them instead at https://storage.cloud.google.com/open-reaction-database/editor/test/protobuf_1dae8fdd.tgz.

Now hopefully someone else can update the GitHub CI testing to unpack that archive in the "editor" directory before running "make test".